tools/update-tool-cron.sh

```diff
@@ -138,6 +141,10 @@ for DOMAIN in * ; do
     popd
 done
 
+if [ "$FAILED_PACKAGES" != "" ]; then
+  echo "WARNING: y2makepot failed for these packages: $FAILED_PACKAGES"
+fi
```

Comment:
I do not like this behavior. As I said, this should return non zero exit and it should be ERROR and not warning. otherwise it looks green, when it is in fact red.